test/jasmine/tests/gl2d_plot_interact_test.js

```diff
@@ -21,7 +21,7 @@ describe('Test removal of gl contexts', function() {
     var gd;
 
     beforeEach(function() {
-        jasmine.DEFAULT_TIMEOUT_INTERVAL = 5000;
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
We may try reverting these (in the end) if we could maintain the performance.

Reply:
Oh yes! Probably these can be reverted, I doubled them while trying to debug something and forgot to put them 
back.

Reply:
@nicholas-esterer Great! Wanna revert these?

Reply:
In the end they can't be reverted, the test fail in that case.